File: Darts/src/code/screen/AboutDialogDarts.java
package code.screen;

import java.awt.Window;

import screen.AbstractAboutDialog;
import util.OnlineConstants;

public class AboutDialogDarts extends AbstractAboutDialog
{
	@Override
	public String getProductDesc()
	{
		return "Darts " + OnlineConstants.DARTS_VERSION_NUMBER;
	}

	@Override
	public Window getChangeLog()
	{
		return new ChangeLog();
	}

}

File: src/main/kotlin/dartzee/screen/AboutDialogDarts.kt
package dartzee.screen

import dartzee.core.screen.AbstractAboutDialog
import dartzee.utils.DARTS_VERSION_NUMBER

class AboutDialogDarts : AbstractAboutDialog()
{
    override fun getProductDesc() = "Darts $DARTS_VERSION_NUMBER"
    override fun getChangeLog() = ChangeLog()
}
